frontend/src/components/teacher/student-management/classroom-summary/DeleteClassroomModal.tsx

```diff
@@ -13,19 +13,14 @@ const DeleteClassroomModal = ({
   onClose,
   deleteClassroom,
 }: DeleteModalProps): React.ReactElement => {
-  const onDeleteClassroom = async () => {
-    await deleteClassroom();
-    onClose();
```

Comment:
I think we still need this `onClose()` - right now the modal doesn't close after deletion

Reply:
hmm, it closes for me @carissa-tang ?

Reply:
![Aug-28-2023 00-17-57](https://github.com/uwblueprint/jump-math/assets/9922514/8e85d962-4f97-45b9-adcf-39ffb298e65a)